modules/perception/lidar/lib/classifier/fused_classifier/ccrf_type_fusion.h

```diff
@@ -18,14 +18,15 @@
 #include <map>
 #include <memory>
 #include <string>
-#include <utility>
 #include <vector>
 
-#include "Eigen/StdVector"
-
+#include "modules/common/util/eigen_defs.h"
 #include "modules/perception/lidar/lib/classifier/fused_classifier/type_fusion_interface.h"
 #include "modules/perception/lidar/lib/classifier/fused_classifier/util.h"
 
+using apollo::common::EigenMap;
+using apollo::common::EigenVector;
```

Comment:
Avoid `using` in header files, esp. in global namespace.

Reply:
Agree with @storypku

Reply:
Such `using` has been removed in a new commit.